cubic-server/Player.cpp

```diff
@@ -71,6 +72,20 @@ Player::Player(std::weak_ptr<Client> cli, std::shared_ptr<Dimension> dim, u128 u
     this->setOperator(Server::getInstance()->permissions.isOperator(username));
     this->_inventory->playerInventory().at(12) = protocol::Slot {true, 734, 42};
     this->_inventory->playerInventory().at(13) = protocol::Slot {true, 1, 12};
+
+    // {display:{Name:'[{"text":"Cubic","italic":false}]'}}
+    constexpr std::string_view NAME = "[{\"text\":\"Cubic\",\"italic\":false}]";
+    constexpr std::string_view NAME_TAG = "Name";
+    constexpr std::string_view DISPLAY_TAG = "display";
+    auto root = nbt_new_tag_compound();
+    auto display = nbt_new_tag_compound();
+    auto name = nbt_new_tag_string(NAME.data(), NAME.size());
+    nbt_set_tag_name(name, NAME_TAG.data(), NAME_TAG.size());
+    nbt_set_tag_name(display, DISPLAY_TAG.data(), DISPLAY_TAG.size());
+    nbt_tag_compound_append(display, name);
+    nbt_tag_compound_append(root, display);
```

Comment:
is there an easier way to create nbt tags ? cause creating 10 variables just for one tags ... :neutral_face:
(maybe something with a json parser could be cool :upside_down_face:)
`this->_inventory->playerInventory().at(14) = protocol::Slot {true, 1, 12, nbt_tag_from_json_string("{display:{Name:'[{\"text\":\"Cubic\",\"italic\":false}]'}}")}`

Reply:
For now it will stay like this, but an nbt json parsing is required if we want to parse some commands from mc.
So yes, this will happen, but when I have the time to make a cpp wrapper for the nbt lib